sktime/forecasting/compose/_hierarchy_ensemble.py

```diff
@@ -159,6 +159,23 @@ def _aggregate(self, y):
 
         return Aggregator().fit_transform(y)
 
+    @property
+    def fitted_list(self):
+        """Deprecated attribute.
+
+        TODO 0.37.0: Remove this property entirely.
+        """
+        import warnings
+
+        warnings.warn(
```

Comment:
this should use `sktime.utils.warnings.warn`, so users can turn it off via `set_config`.